config.xml

```diff
@@ -13,7 +13,13 @@
   <projectNamingStrategy class="jenkins.model.ProjectNamingStrategy$DefaultProjectNamingStrategy"/>
   <workspaceDir>${ITEM_ROOTDIR}/workspace</workspaceDir>
   <buildsDir>${ITEM_ROOTDIR}/builds</buildsDir>
-  <jdks/>
+  <jdks>
+    <jdk>
+      <name>Java 7</name>
+      <home>/Library/Java/JavaVirtualMachines/jdk1.7.0_51.jdk/Contents/Home</home>
```

Comment:
I don't believe this path to jdk will work for our nodes outside of our desktops.